WultraMobileTokenSDK/Operations/Model/UserOperation/WMTUserOperation.swift

```diff
@@ -53,4 +53,10 @@ open class WMTUserOperation: WMTOperation, Codable {
     /// tapping an approve button. If the operation requires 2FA, this value also hints if
     /// the user may use the biometry, or if a password is required.
     public let allowedSignatureType: WMTAllowedOperationSignature
+    
+    /// UI data to be shown
```

Comment:
This is misleading, because some data won't be shown (like blockOnCall), so it should be something like "Data for the operation UI presented".